components/autofill/src/sync/tests/test_reconcile.rs

```diff
@@ -307,6 +327,11 @@ lazy_static::lazy_static! {
             "given-name": "Kip",
             "family-name": "Jones",
         },
+        // Because our record has been "forked" the local change we send out
+        // should have the ORIGINAL unknown fields
+        "outgoing": {
```

Comment:
Let's add `"given-name": "Skip",` to this object, just as an extra test that the record which is outgoing is what we think it is.

The other thing this test should do is somehow check that `reconciled` now has `new-unknown-field` in its mirror, which isn't as simple as adding it to that json. So it's not immediately clear how to get this test to check that :( Maybe something like a new `reconciled_unknown` object or something, and the test has to actually hit the db for the mirror record or something?